src/stylesheets/header.scss

```diff
@@ -23,6 +23,21 @@ body {
 ****** Navbar ******
 ********************/
 
+.navbar-nonProduction {
+  background: var(--color-red-bonita) !important;
```

Comment:
I would prefer if you could not use !important and put this rule at the right place. The use of !important badly impact the maintainability of a project, since the next guy that will work on this component might need to overwrite it and won't be able to do so.